### DISCUSSION TITLE
Customizing Wazuh to the bone

### DISCUSSION THREAD

**Discussion by berrybear98** [wazuh/wazuh#26708] [Q&A]
Hello everybody,
I would customize Wazuh in all the parts.
From the dashboard to the service name of agent.
Do you know if that's possible? 
I would rename the service name in activity manager of pc agent, the app's name installed on it, and all this things.

Thanks for all the answers.

**Comment by MiguelazoDS** [comment]
Hi @berrybear98, 

Mmm you can do it, being open source project you can edit most of the code, but there's no a "setting" to do that. The dashboard may be the easiest one, you just need to find the images that are displayed and replace them. 

Other modifications are not that easy, in fact, we still have our installation folder name as "ossec". You can change the services name and change everywhere is required in the code to make it work with your custom names.

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if needed
